kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/GcpClientBuilder.java

```diff
@@ -59,11 +59,25 @@ public enum KeySource {
   private KeySource keySource = null;
   private String key = null;
 
+  private boolean useStorageWriteApi = false;
+
+  // Scope list taken from : https://developers.google.com/identity/protocols/oauth2/scopes#bigquery
+  private Collection<String> scopes = Lists.newArrayList(
```

Comment:
Looks like a constant variable - can this variable be declared as `static final`?

Reply:
yes, will update.